extensions/notebook/src/test/book/book.test.ts

```diff
@@ -12,91 +12,140 @@ import * as rimraf from 'rimraf';
 import * as os from 'os';
 import { BookTreeViewProvider } from '../../book/bookTreeView';
 import { BookTreeItem } from '../../book/bookTreeItem';
-import * as assert from 'assert';
 
 const SEED = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
 
+export interface ExpectedBookItem {
```

Comment:
Can we re-use an existing interface here?

Reply:
the existing BookTreeItem interface is slightly different and doesn't have nextUri/prevUri properties